### ISSUE TITLE
Resetting filter

### ISSUE THREAD

**Issue by Cortog** [dstrigl/ebusd-config-brink-renovent-excellent-300#9]
Hello,

Thanks a lot for this config file! It works great for my Brink renovent sky 300.
I have one issue though, I can't figure out how to reset the filter. I did enable the option in the csv file by uncommenting it. When I run 
`ebusctl w -c kwl resetfilter`
I get 512 in my mqtt as a return, and the FilterStatus stays on dirty.
I don't have a wall panel to see what it would send to reset the filter. Can anyone help me with this?

**Comment by cibernox** [comment]
Same problem here. I'm not sure what command I should be sending (or rather, what value)

**Comment by pvyleta** [comment]
@cibernox I have the same unit, and I did not yet have the possibility to test my hypothesis, but after reading through decompiled Brink Service Tool, it seems to me that the first byte in the ID should be the slave address of your unit. The files in this repo are designed mainly for Renovent Excelent 400 which has slave address `7c`, but Renovent Sky has slave address of `3c`. Therefore, this command should work for you:

```
w,Sky300,ResetNotifications,,,,4091,3c,,,UIR,0x0001=Errors;0x0100=Filter,,
```

This definition does not allow you to read the response, but it should be possible to be sent through MQTT integration in Home Assistant, which is ultimate goal for me (But I am still not there)

**Comment by cibernox** [comment]
@pvyleta thanks! I have a wolf cwl 300 excellent, so you way the renovent counterpart is the sky300? I don't know if I can test it at the moment because right now I don't have any notification

**Comment by pvyleta** [comment]
So I have good news and bad news. The bad news is I was wrong. The good news is I found the solution: instead of slave address, it has to be zero. I just successfully reset my filter like that. The command I used is this one:

```
w,,ResetNotifications,,,,4091,00,,,UIR,0x0001=Errors;0x0100=Filter,,
```

**Comment by pvyleta** [comment]
At least for my latest SW version of Sky 300, I was able to reset the filter even when the warning was not up, and I confirmed it via the `DaysWithFilter` parameter.

**Comment by cibernox** [comment]
that's great. Can you show how you made the request? I'm assuming you used this screen, right?
![Uploading Screenshot 2024-03-21 at 00.30.15.jpg…]()

**Comment by cibernox** [comment]
@pvyleta that's great! Can you share how you send that message? I'm assuming you used this screen, right?

![Screenshot 2024-03-21 at 00 30 15](https://github.com/dstrigl/ebusd-config-brink-renovent-excellent-300/assets/265339/e29d52f1-4b6a-4f4b-8a4c-a4ec5be04ce1)

**Comment by pvyleta** [comment]
@cibernox Nope, I went to Settings -> Devices -> MQTT and there I found the device (the default name is `ebusd kwl`) and there is the list of the messages you can write:

![image](https://github.com/dstrigl/ebusd-config-brink-renovent-excellent-300/assets/5851137/f64b8549-afd3-42a9-8ac8-e55cd7ac04bc)

**Comment by cibernox** [comment]
@pvyleta did you make any change in yaml files to get those? I don't get any of those fields. In fact I didn't get any field unless I created an entity explicitly in my `config.yaml` file.

**Comment by pvyleta** [comment]
hmmm. no changes in yaml. But I had to use custom `[mqtt-hassio.cfg](https://github.com/john30/ebusd/blob/master/contrib/etc/ebusd/mqtt-hassio.cfg)` through the configuration of the add-on, and enable write operations there.

The entity was created automatically by ebusd addon.

How does your setup looks like? I am assuming HA + ebusd addon? Can you post what the commandline looks like when lauching the addon? You can see that as one of the first lines in Log tab of the addon, after you restart the addon.

**Comment by tiotuba** [comment]
> So I have good news and bad news. The bad news is I was wrong. The good news is I found the solution: instead of slave address, it has to be zero. I just successfully reset my filter like that. The command I used is this one:
> 
> ```
> w,,ResetNotifications,,,,4091,00,,,UIR,0x0001=Errors;0x0100=Filter,,
> ```

hi @pvyleta, this worked great for me! After setting it to Filter I confirmed it like you did with DaysWithFilter.

**Comment by badi19** [comment]
This worked for me:
Config ebusd to use mqtt-hassio.config. see link in a message above.
In the mqtt-hassio.config I added search strings to this option:
filter-name = status|temp|humidity|yield|count|energy

For example reset|filter|actual|flow|bypass. A whole bunch of sensors become available in homeassistant after this.
Also make sure to uncomment this line to be able to change fanmode, notifications etc.:
filter-direction = r|u|^w
